### PR TITLE
Cleanup warnings

### PR DESCRIPTION
**Pull request by tips48**
I cleaned up most of the warnings in all the Spout projects. It was mainly imports, but there were some issues with not closing resources and type casting.
Enjoy!

### CODE REVIEW

File: src/main/java/org/spout/engine/listener/SpoutProxyConnectListener.java [hunk 1]
```diff
@@ -41,7 +40,8 @@
 
 public class SpoutProxyConnectListener implements ChannelFutureListener {
 	
-	private final Engine engine;
+	@SuppressWarnings("unused")
+        private final Engine engine;
```
Comment (Raphfrk) [inline src/main/java/org/spout/engine/listener/SpoutProxyConnectListener.java:44]: Is that correct formatting?

Comment (Afforess) [inline src/main/java/org/spout/engine/listener/SpoutProxyConnectListener.java:44]: Nope. engine should not be indented.

Comment (tips48) [inline src/main/java/org/spout/engine/listener/SpoutProxyConnectListener.java:44]: My client derped whenever I added a @SurpressWarnings and indented the next line..I'll fix next time I'm free

File: src/main/java/org/spout/engine/batcher/GuiRenderer.java [hunk 2]
```diff
@@ -50,12 +47,14 @@ public static void init(){
 		
 	}
 	
-	private static void TexturedQuad(float x, float y, float width, float height){
+	@SuppressWarnings("unused")
+        private static void TexturedQuad(float x, float y, float width, float height){
```
Comment (olliestanley) [inline src/main/java/org/spout/engine/batcher/GuiRenderer.java:51]: Just so you don't miss it, formatting here as well

**Pull request closed without merging**